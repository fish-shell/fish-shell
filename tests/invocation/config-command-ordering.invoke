-c "echo command" -C "echo init-command"
