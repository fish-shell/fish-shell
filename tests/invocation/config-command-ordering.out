in config.fish
init-command
command
